deploy/split-infra-tenant/040-controller.yaml

```diff
@@ -0,0 +1,122 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kubevirt-csi-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubevirt-csi-driver
+  template:
+    metadata:
+      labels:
+        app: kubevirt-csi-driver
+    spec:
+      hostNetwork: true
```

Comment:
do we really need `hostNetwork: true`?

Reply:
Ha, i didn't catch that. Great question. I copied these over from the other deploy scripts and just made a few modifications here and there. I'll try taking this hostNetwork part out and see what happens.

Reply:
fixed, we don't need hostNetwork. I think this was left over from copying manifests from the aws csi driver's yaml.